features/bevragen/persoon/gezag/gba/fields-gba.feature

```diff
@@ -0,0 +1,155 @@
+      | 000000036       | GG         | 000000048        |
+
+
+  Rule: Het vragen van één of meerdere velden van 'gezag' levert alle velden van 'gezag' die van toepassing zijn op de persoon
+
+    Scenario: gezag vragen met fields <fields> geeft alle van velden van alle soorten gezag van toepassing op de persoon
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | <fields>                        |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde                   |
+      | type                             | EenhoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000012                |
+      | ouder.burgerservicenummer        | 000000048                |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000024                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000048 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000061 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036        |
+      | ouder.burgerservicenummer        | 000000048        |
+      | derde.burgerservicenummer        | 000000061        |
+
+      Voorbeelden:
+      | fields                                     |
+      | gezag                                      |
+      | gezag.type                                 |
+      | gezag.minderjarige                         |
+      | gezag.ouders                               |
+      | gezag.ouder                                |
+      | gezag.derde                                |
+      | gezag.derden                               |
+      | gezag.minderjarige.burgerservicenummer     |
+      | gezag.ouders.burgerservicenummer           |
+      | gezag.ouder.burgerservicenummer            |
+      | gezag.derde.burgerservicenummer            |
+      | gezag.derden.burgerservicenummer           |
+      | gezag.ouders,gezag.ouder                   |
+      | gezag.type,gezag.minderjarige,gezag.ouders |
+
+    Scenario: gezag vragen met fields <fields> dat niet van toepassing is op de persoon geeft alle velden van gezag die wel van toepassing zijn op de persoon
```

Comment:
dit is toch een subset van de vragen-met-fields hierboven? Wordt dit scenario niet al door bovenstaand scenario gedekt?

Reply:
niet echt, want in vorige scenario komen alle gevraagde velden voor in het antwoord